GNUmakefile

```diff
@@ -2,13 +2,24 @@
 
 default: build
 
-ifneq ($(PRODUCT),)
+# mm setup
+ifeq ($(ENGINE),tpgtools)
+  mmv1_compile=-p blahblah
```

Comment:
Instead of `blahblah` can we do something more descriptive like `does-not-exist`? Also, can we document inline why we're providing dummy products instead of skipping the tool entirely?

Reply:
ah good point, :]